### PR TITLE
style: issue #302 refine tree tag component

### LINKED ISSUE #302: Refine the UI/layout of the tree tag component
Screen Shot 2021-11-26 at 11 32 27 AM

The design: 

image

I guess this is about making the component responsive.
---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by VWRoli**
# Description

Refine tree tag component styling to match the [Figma](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=2497%3A9322) design.

Fixes #302

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Cypress component tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by QuaidBartolomei — CHANGES_REQUESTED** [review]
2021-12-07 22 44 21 localhost 1140190ada17
Here's a screenshot. The letter spacing looks a little tight

**Comment by VWRoli** [comment]
Added the letter spacing.

File: src/components/common/TreeTag.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    padding: theme.spacing(4),
+    color: theme.palette.common.white,
+    background: theme.palette.primary.main,
+    borderRadius: theme.spacing(4),
+    width: 'fit-content',
+    letterSpacing: '0.02em',
+  },
+}));
+
+function TreeTag({ TreeTag, title, icon }) {
```
Comment (QuaidBartolomei) [inline src/components/common/TreeTag.js:21]: rename TreeTag prop to fix lint warning:
`'TreeTag' is already declared in the upper scope on line 21 column 10.eslint (no-shadow)`

File: src/components/common/TreeTag.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const { classes } = useStyles();
+  return (
+    <Grid container className={classes.container}>
+      <Grid item sx={{ pr: 4 }}>
+        {icon}
+      </Grid>
+      <Grid item>
+        <Typography variant="body1">{title}</Typography>
+        <Typography variant="h5">{TreeTag}</Typography>
```
Comment (QuaidBartolomei) [inline src/components/common/TreeTag.js:30]:
```suggestion
        <Typography variant="h5"
        sx={{
          fontFamily: 'Lato',
          fontWeight: 800
        }}
        >{TreeTag}</Typography>
```

**Review by QuaidBartolomei — CHANGES_REQUESTED** [review]
2021-12-08 01 27 44 localhost 4f4587c36846
Here's an updated screenshot, I think the icon shouldn't wrap to the top like the latitude/longitude one is. I also noticed the Figma document was using a different font which is why the spacing still seems different.

File: src/components/common/TreeTag.js (hunk 1 above)
Comment (VWRoli) [reply src/components/common/TreeTag.js]: Renamed the TreeTag prop to TreeTagValue.
Added a `nowrap` property to the CSS.
The font will be fixed once the branch is merged, as this branch was created before #301. That will fix this issue.

Screenshot from 2021-12-08 11-27-15

**Comment by dadiorchen** [comment]
@QuaidBartolomei I want to ask how do you check the code and take the screenshot? by download and run it? I was thinking that maybe we can do some automatic screenshots and video records by cypress, there is a feature by Cypress for this purpose, so we can easily check the changing of things without bothering to run it? 
what do you guys think?

**Comment by QuaidBartolomei** [comment]
@dadiorchen Currently I am pulling the branch and running the code locally. Cypress as a `cy.screenshot()` command that can be used in any cypress tests. I think the command will need to be manually added to the test file though. Videos are always recorded for integration tests by default. Once we add cypress to the CI workflow we can also save the screenshots and videos from the CI test runs.

**Review by QuaidBartolomei — APPROVED** [review]

**Comment by dadiorchen** [comment]
@QuaidBartolomei so we can check the video or screenshot after the CI running? I'm wondering where could we find those files on CI?

**Pull request merged**